codemirror/codemirror-tests.ts

```diff
@@ -39,10 +39,7 @@ var annotation: CodeMirror.Annotation = {
         ch: 0,
         line: 0
     },
-    to: {
-        ch: 1,
-        line: 0
-    },
+    to: CodeMirror.Pos(1),
```

Comment:
They need to be Pos arguments, the latter tests the PositionConstructor. I can make them separate tests if you'd prefer.